pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -309,7 +300,12 @@ func (f *ring1Factory) Scaler(mapping *meta.RESTMapping) (kubectl.Scaler, error)
 func (f *ring1Factory) Reaper(mapping *meta.RESTMapping) (kubectl.Reaper, error) {
```

Comment:
@pwittrock @adohe @soltysh I think we need to start trying to kill reapers.  GC doesn't resolve problems for objects that are nested (remove this SA subject from that rolebinding), but we should start thinking of alternatives.